idaes/generic_models/properties/core/eos/tests/test_enrtl_verification_NaCl.py

```diff
@@ -46,21 +47,28 @@
     relative_permittivity_constant
 
 
-def dummy_method(b, *args, **kwargs):
-    return 1000/18e-3*pyunits.mol/pyunits.m**3
+class ConstantVolMol():
+    def build_parameters(b):
+        b.vol_mol_pure = Param(initialize=18e-6,
+                               units=pyunits.m**3/pyunits.mol,
+                               mutable=True)
+
+    def return_expression(b, cobj, T):
+        return cobj.vol_mol_pure
 
 
 configuration = {
     "components": {
         "H2O": {"type": Solvent,
-                "dens_mol_liq_comp": dummy_method,
+                "vol_mol_liq_comp": ConstantVolMol,
```

Comment:
Oh ok this answers my previous question on the naming. How bad is it to change this so _phase is in the name and the variable is indexed by the phase?

Reply:
Again - these are pure component properties, not mixture properties, and the name is different to make that clear. Also, these apply to *any* liquid phase, i.e. there will only be one of these no matter how many liquid phases there are in the mixture.